src/main/java/io/vertx/httpproxy/interceptors/PathInterceptor.java

```diff
@@ -11,19 +11,46 @@
 
 package io.vertx.httpproxy.interceptors;
 
+import io.vertx.codegen.annotations.Unstable;
 import io.vertx.codegen.annotations.VertxGen;
 import io.vertx.httpproxy.ProxyInterceptor;
 import io.vertx.httpproxy.interceptors.impl.AddPrefixPathInterceptorImpl;
+import io.vertx.httpproxy.interceptors.impl.PathInterceptorImpl;
 import io.vertx.httpproxy.interceptors.impl.RemovePrefixPathInterceptorImpl;
 
+import java.util.function.Function;
+
 @VertxGen
+@Unstable
```

Comment:
In addition to the Javadoc for the methods, can you please also add Javadoc for the class?